### ISSUE TITLE
Allow contributors to self-assign issues with GitHub workflow

### ISSUE THREAD

**Issue by sugatoray** [goodmami/wn#185]
Commenting `#self-assign` in any issue will let repository members to self-assign that issue to themselves.

:point_right: This reduces chores for repo-maintainer(s).

**Comment by sugatoray** [comment]
cc: @goodmami 

I have already sent a PR for adding this functionality. Please assign this to me. 

> 🔔 If/once the PR is merged, then in future, issue assignment can be done by contributors themselves with `#self-assign` comment alone.

- PR: #186   

> 👉 Like this: **#self-assign**

**NOTE**: 

*I have also raised another issue #184 to add `wn` to conda-forge channel and enable conda driven installation. I was not able to self-assign then. I have earlier added this workflow to other repositories as well. Please see the following PR:* 

- https://github.com/unionai-oss/unionml/pull/136

**Comment by goodmami** [comment]
My first thought is that Wn doesn't yet have an active enough contributor pool for this solution to address a real problem. But my second thought is that it doesn't seem like a big change or commitment, so why not? I also don't particularly like macro-triggering comments cluttering issue threads, but that's minor.

Regarding the implementation though, firing up a docker container just to send a curl command to set the assignees seems heavy. Why not use a [Javascript action](https://docs.github.com/en/actions/creating-actions/about-custom-actions#javascript-actions) with [github-script](https://github.com/actions/github-script)? It would be faster, at least.

**Comment by goodmami** [comment]
@sugatoray thanks for raising the issue and making the PR, but I don't think this is needed at this moment. The project doesn't yet have enough activity that it solves any real need. And I don't mind assigning issues when people request them. The manual step also provides the chance to have some conversation about the issue first.
